src/components/common/deleteResource.jsx

```diff
@@ -63,10 +64,12 @@ export class DeleteResourceModal extends React.Component {
         return (
             <Modal position="top" variant="small" isOpen onClose={onClose}
                    id="delete-resource-modal"
-                   title={title}
+                   title={<>
```

Comment:
Modal has `titleIconVariant` property where you can specify either `warning` or specific icon, see https://www.patternfly.org/v4/components/modal

Cannot we use that instead?

Reply:
Good catch, thanks!